website/addons/googledrive/utils.py

```diff
@@ -5,9 +5,8 @@
 import logging
 from urllib import quote
 
-from website.util import web_url_for
+from website.addons.citations import utils
```

Comment:
If you need this import, please move the needed utility functions to website/addons/base/utils.py and import from there.